src/test/java/com/wikia/webdriver/common/testnglisteners/InvokeMethodAdapter.java

```diff
@@ -22,6 +22,7 @@ public void afterInvocation(IInvokedMethod method, ITestResult result) {
         result.setStatus(ITestResult.FAILURE);
         result.setThrowable(new TestFailedException());
       }
+      //todo transofrm result into better exception
       if (verificationFailures.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)